queue/queue_test.go

```diff
@@ -782,7 +782,7 @@ func TestUtube_Put(t *testing.T) {
 			return
 		}
 
-		time.Sleep(2 * time.Second)
+		time.Sleep(2*time.Second + 30*time.Millisecond)
```

Comment:
Such approach is definitely not good.

Reply:
I'll try to explain:

A situation that is solved by increasing the waiting time by a bit looks like a data race. Such a fix should be done only if you are absolutely sure that this is the expected behavior, and not a data race (and it is advisable to add a comment on this).

Reply:
I understand it, it was done for experimentational purposes on github runners. I will not use it or give a full explanation for it.